projects/Mallard/src/helpers/push-download-failsafe.ts

```diff
@@ -0,0 +1,39 @@
+import BackgroundFetch from 'react-native-background-fetch'
+import { pushTracking } from './push-tracking'
+import { clearAndDownloadIssue } from './download-todays-issue'
+
+const pushDownloadFailsafe = () => {
+    BackgroundFetch.configure(
+        {
+            minimumFetchInterval: 120, // Every 2 hours
+            stopOnTerminate: false,
+            startOnBoot: true,
+        },
+        async () => {
+            await pushTracking('backgroundFetch', 'started')
+            await clearAndDownloadIssue()
+            await pushTracking('backgroundFetch', 'ended')
+            BackgroundFetch.finish(BackgroundFetch.FETCH_RESULT_NEW_DATA)
```

Comment:
does it mean the callaback won't get called again after 1 successful attempt? what about the next day?

Reply:
It gets called every time unless there is an error. This is required to stop the OS from considering you a nuisance. See the example code here: https://github.com/transistorsoft/react-native-background-fetch#config-boolean-enableheadless-false